tests/suites/0_stateless/20+_others/20_0001_planner_v2.sql

```diff
@@ -142,6 +142,8 @@ create table t3(a int, b int);
 insert into t3 values(1,2),(2,3);
 select * from t3 order by 2 desc;
 select a from t3 order by 1 desc;
+select a from t3 order by t3.a + 1;
+select number from numbers(10) order by number + 1;
```

Comment:
`numbers(10) `can be simplify to `numbers(3)`

Need cover:
`select number + 3 as c from numbers(3) order by c + 1, -number`
`select number, avg(number) c from numbers(3) group by number order by c + 1`

Reply:
covered